src/utils.py

```diff
@@ -962,6 +993,22 @@ def dijkstra(self, queue: List[Tuple[float, float, RelaxedFact]]) -> None:
                 fact.expanded = True
 
 
+class HAddHeuristic(_RelaxationHeuristic):
```

Comment:
Let's make HAddHeuristic and HMaxHeuristic private so create_heuristic is the only interface